mu/app.py

```diff
@@ -104,9 +104,10 @@ def run():
     # Setup the window.
     editor_window.closeEvent = editor.quit
     editor_window.setup(editor.debug_toggle_breakpoint, editor.theme)
-    # capture the filename passed by the os, if there was one
-    passed_filename = sys.argv[1] if len(sys.argv) > 1 else None
-    editor.restore_session(os.path.abspath(passed_filename))
+    # Restore the previous session
+    editor.restore_session(without_empty=len(sys.argv[1:]) > 0)
```

Comment:
Hmmm... so why pass this flag at all. Why not pass in `sys.argv[1:]` and let `restore_session` handle it..? Happy to be talked round, but it feels like there should be one call to `restore_session` with the passed in files and then it'll work out what to do with it given the previous state of the session and the new files. Does that make sense?

Reply:
I had them separate in case we wanted to do single instance things but I'll move the call inside `restore_session`